packages/picasso/src/Dropdown/Dropdown.tsx

```diff
@@ -223,10 +226,10 @@ export const Dropdown = forwardRef<HTMLDivElement, Props>(function Dropdown(
         {typeof children === 'function' ? children({ isOpen }) : children}
       </div>
 
-      {anchorEl && isOpen && (
+      {((!keepMounted && isOpen) || keepMounted) && (
```

Comment:
```suggestion
      {(isOpen || keepMounted) && (
```

Shouldn't this be enough? 🤔

Reply:
why don't need the `anchorEl` anymore? 🤔

Reply:
The `anchorEl` gets reset when we close the menu
```
  const forceClose = () => {
    setAnchorEl(undefined)
    setIsOpen(false)
    onClose()
  }
```

Tested with via `@toptal/picasso@24.0.1-alpha-fix-for-disable-portal-v2.1`

Reply:
Found a bug with it, will convert to draft for now

![image](https://user-images.githubusercontent.com/79158829/167810013-3d6d69d4-bc45-44e5-b43c-c0ce3ccfad67.png)

Reply:
Fixed 8e2e94a